enclave-tls/CMakeLists.txt

```diff
@@ -0,0 +1,93 @@
+# CMake version
```

Comment:
I think also need to add copyright info to each new added CMAKE related .txt and .cmake files.

Reply:
Yanrong will do this by some scripts.